fpnge.cc

```diff
@@ -955,34 +1036,34 @@ size_t FPNGEEncode(size_t bytes_per_channel, size_t num_channels,
   // allows for padding, and for extra initial space for the "left" pixel for
   // predictors.
   size_t bytes_per_line_buf =
-      (bytes_per_line + 4 * bytes_per_channel + 31) / 32 * 32;
+      (bytes_per_line + 4 * bytes_per_channel + SIMD_WIDTH-1) / SIMD_WIDTH * SIMD_WIDTH;
```

Comment:
Can you run clang-format?

Reply:
Looks like I missed the spacing there. Haven't used clang-format in the past, so what settings/arguments do you recommend?

Reply:
I'd just use it with default settings and in-place modification, i.e. `clang-format -i fpnge.cc`.